setup.cfg

```diff
@@ -38,6 +38,7 @@ setup_requires =
 # The others are in files/install-deps*.yaml
 install_requires =
     celery[redis]
+    marshmallow~=2.0
```

Comment:
Actually we are not forced to use v2 in `packit-service` because we don't package it into Fedora.
You are free to use v3  :-)